File: Java/Sample005.java
import java.util.Random;

public class Sample005 {
    public static void main(String[] args) {
        Random random = new Random();

        int money = random.nextInt(100);
        int love = random.nextInt(100);
        int health = random.nextInt(100);
        int total = (money + love + health) / 3;

        System.out.println("金運: "+money);
        System.out.println("恋愛運: "+love);
        System.out.println("健康運: "+health);
        System.out.println("総合運: "+total);
    }
}

File: Python/sample005.py
import random

money = random.randint(1, 100)
love = random.randint(1, 100)
health = random.randint(1, 100)
total = int((money + love + health) / 3)

print(f"金　運: {money}")
print(f"恋愛運: {love}")
print(f"健康運: {health}")
print(f"総合運: {total}")
